lib/document_view.dart

```diff
@@ -14,14 +14,30 @@ class DocumentView extends StatefulWidget {
 class _DocumentViewState extends State<DocumentView> {
   @override
   Widget build(BuildContext context) {
+  final String viewType = 'pdftron_flutter/documentview';
+
     if (Platform.isAndroid) {
-      return AndroidView(
-        viewType: 'pdftron_flutter/documentview',
-        onPlatformViewCreated: _onPlatformViewCreated,
-      );
+      return PlatformViewLink(
+        viewType: viewType,
+        surfaceFactory: (BuildContext context, PlatformViewController controller) {
+          return AndroidViewSurface(
+            controller: controller as AndroidViewController, 
+            hitTestBehavior: PlatformViewHitTestBehavior.opaque, 
+            gestureRecognizers: const <Factory<OneSequenceGestureRecognizer>>[].toSet());
+        }, 
+        onCreatePlatformView: (PlatformViewCreationParams params) {
+          return PlatformViewsService.initSurfaceAndroidView(
+            id: params.id,
+            viewType: viewType,
+            layoutDirection: TextDirection.ltr,
```

Comment:
if hard code ltr here, what if user runs a RTL device?

Reply:
The API Docs for `initSurfaceAndroidView` states that `layoutDirection` cannot be null. There are three constants we can pass `TextDirection.ltr`, `TextDirection.rtl` and `TextDirection.values` (the last constant is described as a "constant List of the values in this enum, in order of their declaration"). Would it be best to use `TextDirection.values`?

Link to `initSurfaceAndroidView`: https://api.flutter.dev/flutter/services/PlatformViewsService/initSurfaceAndroidView.html
Link to the `TextDirection` enum: https://api.flutter.dev/flutter/dart-ui/TextDirection-class.html